### DISCUSSION TITLE
Is there any way to create Sales through the API?

### DISCUSSION THREAD

**Discussion by GorgonFreeman** [etsy/open-api#1071] [Ideas]
I'd love to set up an automated system around sales, e.g. to make one every day with a few random products, but can't see anything in the docs.

**Comment by emergingdzns** [comment]
No. Sales are only allowed on Etsy.com or in the Etsy app. No 3rd party access.

**Reply by GorgonFreeman** [reply]
Thanks for the info, then my suggestion is that it become supported. I'm only trying to do it for my own store, so I don't see any reason that would be excluded.

**Reply by Archieversam** [reply]
Are you looking for a way to track and convert majority of your traffic into potential buyers in an easy way? Kindly contact us now for more details or use the link below. THANKS.
https://cutt.ly/Sw0vSP70

**Comment by Archieversam** [comment]
Are you looking for a way to track and convert majority of your traffic into potential buyers in an easy way? Kindly contact us now for more details or use the link below. THANKS.
https://cutt.ly/Sw0vSP70
